tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -158,7 +158,112 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
+    ${new_list_of_tile} =    Get Slice From List    ${list_of_all_items_id}    0    ${range}
+    Lists Should Be Equal    ${new_list_of_tile}    ${list_of_fav_items_id}    ignore_order=${True}
+
+Verify Star Icons Are Clickable
+    [Documentation]    Verifies that star icons in the resources page are clickable
+    [Arguments]    ${list_of_ids}
+    FOR    ${id}    IN    @{list_of_ids}
+         Set Item As Favourite    ${id}
+         Remove An Item From Favourite    ${id}
+    END
+
+Get List Of Ids Of Tiles
+    [Documentation]    Returns the list of ids of tiles present in resources page
+    ${list_of_ids}=    Get List Of Atrributes
+    ...    xpath=//article[@class="pf-c-card pf-m-selectable odh-card odh-tourable-card"]    attribute=id
+    [Return]    ${list_of_ids}
+
+Set Item As Favourite
+    [Documentation]    Add the tiles in favourite
+    [Arguments]    ${id}
+    ${not_clicked} =    Get Element Attribute    //*[@id="${id}"]/div[1]/span    class
+    Should Be Equal    ${not_clicked}    odh-dashboard__favorite
+    Click Element    //*[@id="${id}"]/div[1]/span
+
+Remove An Item From Favourite
+    [Documentation]    Removes the tiles from favourite
+    [Arguments]    ${id}
+    ${clicked} =    Get Element Attribute    //*[@id="${id}"]/div[1]/span    class
+    Should Be Equal    ${clicked}    odh-dashboard__favorite m-is-favorite
+    Click Element    //*[@id="${id}"]/div[1]/span
+
+Add The Items In Favourites
+    [Documentation]    Add the tiles in the favourites
+    [Arguments]    @{list_of_ids}
+    FOR    ${id}    IN     @{list_of_ids}
+        Set Item As Favourite    ${id}
+    END
+
+Favorite Items Should Be Listed First When Sorted By
+    [Documentation]    Changes the sort type of tile and checks that favourites
+    ...                favourite items are still listed first
+    [Arguments]    ${list_of_ids_of_favourite}    ${sort_type}
+    Sort Resources By    ${sort_type}
+    ${new_list_of_tile} =    Get List Of Ids Of Tiles
+    Favorite Items Should Be Listed First    ${list_of_ids_of_favourite}    ${new_list_of_tile}    ${21}
+
+Get The List Of Ids of Tiles In List View
+    [Documentation]    Returns the list of ids of tiles in list view
+    ${list_of_new_tile_ids} =    Get List Of Atrributes    //div[@class="odh-list-item__doc-title"]    id
+    ${len} =    Get Length    ${list_of_new_tile_ids}
+    ${list_of_ids_in_list_view} =    Create List
+    FOR    ${index}    IN RANGE    0    ${len}    2
+        Append To List    ${list_of_ids_in_list_view}    ${list_of_new_tile_ids}[${index}]
+    END
+    [Return]    ${list_of_ids_in_list_view}
+
+Remove Items From Favourites
+    [Documentation]    Removes the items from favourites
+    [Arguments]    @{list_of_ids}
+    FOR    ${id}    IN     @{list_of_ids}
+        Remove An Item From Favourite    ${id}
+    END
+    Close Browser
+
+Change The Sort
+    [Documentation]    Changes the sort of items in resource page
```

Comment:
could u plz change the doc in an understandable way, its bit confusing for now

Reply:
I have updated doc in new commit